### ISSUE TITLE
How to staticaly link the sciter lib

### ISSUE THREAD

**Issue by albertoVieiraNeto** [sciter-sdk/rust-sciter#121]
hi, i have a license for sciter and already generated the sciter.static.lib, how should i proceed to embed the static library to my final executable? can you just give me a direction?

**Comment by pravic** [comment]
I need to add this to rust-sciter: https://github.com/c-smile/sciter-js-sdk/blob/main/include/sciter-x-api.h#L306-L324

**Comment by pravic** [comment]
Is it always `sciter.static.lib`, on all platforms?

**Comment by albertoVieiraNeto** [comment]
no, on windows by default is ".lib" and linux ".a", on mac i dont know.

**Comment by Yatekii** [comment]
On macOS it is '.dylib'.
